sources/packages/backend/libs/sims-db/src/entities/application.model.ts

```diff
@@ -415,6 +415,19 @@ export class Application extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   applicationEditStatusNote?: Note;
+
+  /**
+   * Application versions that were created from this application
+   * if the application is the parent application.
+   */
+  @OneToMany(
+    () => Application,
+    (versionApplication) => versionApplication.parentApplication,
+    {
+      nullable: true,
+    },
+  )
+  versions?: Application[];
```

Comment:
Since every application starts with a parent I would say that this is not nullable. Does it make sense?

Reply:
If the nullable is due the moment when we filter the application itself, an empty array will be returned if I am not wrong.